README.rst

```diff
@@ -59,8 +59,6 @@ Documentation
 | Find the `documentation <https://open-mastr.readthedocs.io/en/latest/>`_ hosted on ReadTheDocs.
 
 | The original API documentation can be found on the `Webhilfe des Marktstammdatenregisters <https://www.marktstammdatenregister.de/MaStRHilfe/subpages/webdienst.html>`_.
-| If you are interested in browsing the MaStR online, check out the privately hosted `Marktstammdatenregister.dev <https://marktstammdatenregister.dev/>`_.
-| Also see the `bundesAPI/Marktstammdaten-API <https://github.com/bundesAPI/marktstammdaten-api>`_ for another implementation.
```

Comment:
Why did you remove these links? They're are not essential but could be helpful. Maybe we could move them to the docs?  To my mind they do not really fit into the current structure but what so you think about a new section after [Development](https://open-mastr.readthedocs.io/en/latest/development/contributing_mirror/) with helpful links in general?

Reply:
Yes, you are right. I remember removing them because we got a bug report for marktstammdatenregister.dev once, so we might rephrase it that it is clear that this is not from us.

Reply:
Keeping and rephrasing sounds good to me :+1:

Reply:
I just checked and https://marktstammdatenregister.dev seems to be down (not the landing page, but the actual datasette pages where you can click through the dataset). Should we still mention it @nesnoj?

Reply:
Yes, but I'd use the github orga: https://github.com/marktstammdatenregister-dev

(BTW: They have [Zenodo releases of the data](https://zenodo.org/records/10427795) but it no longer seems to be maintained - time for me to boost #559 ;)